app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe):
```

Comment:
This function returns "Friends should buy 1 masks" instead of "All friends should be vaccinated"
for [not_wearing_mask, not_vaccinated] case
```
(
    [
        {
            "name": "Alisa",
            "vaccine": {
                "name": "Pfizer",
                "expiration_date": datetime.date.today(),
            },
            "wearing_a_mask": False,
        },
        {
            "name": "Bob",
            "wearing_a_mask": True,
        },
    ],
    Cafe("KFC"),
    "All friends should be vaccinated",
),
```
Consider this test for the  `test_go_to_the_cafe` function